rcl_logging_spdlog/test/test_logging_interface.cpp

```diff
@@ -62,6 +65,42 @@ class RestoreEnvVar
   const std::string value_;
 };
 
+class TemporaryLogDir final
+{
+public:
+  TemporaryLogDir()
+  : orig_ros_log_dir_value_(rcpputils::get_env_var("ROS_LOG_DIR"))
+  {
+    // Create a random directory in our current path
+    std::random_device dev;
+    std::mt19937 prng(dev());
+    std::uniform_int_distribution<uint64_t> rand(0);
+
+    std::stringstream ss;
+    ss << std::hex << rand(prng);
+
+    local_log_dir_ = std::filesystem::current_path() / ss.str();
+
+    if (!std::filesystem::create_directories(local_log_dir_)) {
+      throw std::runtime_error("Failed to make temporary log directory");
+    }
```

Comment:
You could use [`rcpputils::fs::create_temp_directory()`](https://github.com/ros2/rcpputils/blob/292be119c13edb8baa1e78c348637c460ad7a846/src/filesystem_helper.cpp#L310) for this. Since `std::filesystem` doesn't offer a function like that, it might be kept after removing `rcpputils::fs`.

Reply:
Yeah, that's a good point.  I'll go ahead and do that.

Reply:
Fixed in [cdfb9df](https://github.com/ros2/rcl_logging/pull/115/commits/cdfb9df6eacc84ea5c8e2d5aa97536ac83439bd6)